2016-07-27/1.php

```diff
@@ -19,7 +20,18 @@
  * @return mixed[]
  */
 function arrayReverse($a) {
-    return array_reverse($a);
+    $reversed = [];
+    $arrayCount = 0;
+
+    foreach ($a as $element) {
```

Comment:
As with your added information I saw your solution was using the builtin "count" method.
I removed it for you.